Podverse/EpisodesTableViewController.swift

```diff
@@ -220,6 +220,20 @@ class EpisodesTableViewController: PVViewController, UITableViewDataSource, UITa
         tableView.deselectRow(at: indexPath as IndexPath, animated: true)
 
     }
+    
+    func tableView(_ tableView: UITableView, editActionsForRowAt indexPath: IndexPath) -> [UITableViewRowAction]? {
+        let episodeToEdit = episodesArray[indexPath.row]
+        
+        let deleteAction = UITableViewRowAction(style: .default, title: "Delete", handler: {action, indexpath in
+            self.episodesArray.remove(at: indexPath.row)
+            self.tableView.deleteRows(at: [indexPath], with: .fade)
+            DispatchQueue.global().async {
+                PVDeleter.deleteEpisode(episodeId: episodeToEdit.objectID, shouldCallProtocolMethod: true)
```

Comment:
Maybe dispatch to the background the code inside the deleteEpisode method so you don't have to do it always when you call the method